src/components/dialogs/generator-modification-dialog.js

```diff
@@ -524,12 +524,54 @@ const GeneratorModificationDialog = ({
         defaultValue: getValue(formValues?.stepUpTransformerReactance),
         previousValue: generatorInfos?.stepUpTransformerReactance,
     });
+
+    const [plannedActivePowerSetPoint, plannedActivePowerSetPointField] =
+        useDoubleValue({
+            label: 'PlannedActivePowerSetPoint',
+            adornment: ActivePowerAdornment,
+            inputForm: inputForm,
```

Comment:
Christelle asked me to add "optional" in the label, you have to do like here : https://github.com/gridsuite/gridstudy-app/pull/1047/files

Reply:
We have a ticket to remove "optional" in the label in the generator modification form
https://dev.azure.com/GridSuite/MVP/_sprints/backlog/MVP%20Team/MVP/Sprint%2058?workitem=1737

Reply:
Ah yeah anyway all fields are optional in a modification form... my bad